From b66edbc20b4063d45aaa75191be536c27d137584 Mon Sep 17 00:00:00 2001
From: Vilius Sutkus '89 <Vilius@ViliusSutkus89.com>
Date: Thu, 13 Jun 2024 14:19:32 +0300
Subject: [PATCH 2/4] [CI] Use Ubuntu 22.04, upgrade actions/checkout and
 actions/upload-artifact

Ubuntu 20.04 comes with Freetype 2.10.1, Poppler requires at least 2.11
---
 .github/workflows/build.yml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index f7bd2f8..c1a39cf 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -4,16 +4,16 @@ on: [push]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Build project with APT
         run: './buildScripts/buildInstallLocallyApt'
 
       - name: Upload artifact
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: my-artifact
           path: pdf2htmlEX/build
-- 
2.39.3 (Apple Git-146)

